docs/media-analytics/custom-player.md

```diff
@@ -8,7 +8,7 @@ In this guide we will learn how you can implement [Media Analytics](http://www.m
  
 ## Tracking a new media player in the browser
 
-The Media Analytics plugin currently supports  out of the box the HTML5, Vimeo and YouTube media players. If you are using
+The Media Analytics plugin currently supports out of the box the HTML5, Vimeo and YouTube media players. If you are using
```

Comment:
I had it first here but then removed it again. That's why there is the space change. I'd personally rather not have it here. They are all based on HTML5 and we actually support many more players etc. We can add it, I just prefer not to